vermouth/processors/annotate_mut_mod.py

```diff
@@ -223,7 +225,11 @@ def annotate_modifications(molecule, modifications, mutations):
                                  _format_resname(res), key, mod)
                     for node_idx in res['graph']:
                         molecule.nodes[node_idx][key] = molecule.nodes[node_idx].get(key, []) + [mod]
-
+            if mod_found == False:
+                LOGGER.warning('{} with resid {} not found. '
+                                 'No modification made.'
+                                 ''.format(resspec['resname'], resspec['resid']))
```

Comment:
Small problem actually. Not all resspecs need to have a resname or resid attribute. Might be better to format the respec like the CLI accepts it. You should be able to use `_format_resname` for that. Also that function has a bad bad name. Pretty sure that's my fault though.

Reply:
Thanks for the heads up on `_format_resname`, will update using that.